kustomize/overlays/opendatahub/kustomization.yaml

```diff
@@ -0,0 +1,44 @@
+namespace: opendatahub-operator
+
+
+images:
+- name: aipipeline/kubeflow-operator:v1.0.0
```

Comment:
are we going to have our own images?

Reply:
Yes, that's what the `kustomize edit set image aipipeline/kubeflow-operator:v1.0.0 ${OPERATOR_IMG}` bit in the testing instructions was attempting to set.